### DISCUSSION TITLE
Make BCC work (for now BCC with reverse aliases not supported)

### DISCUSSION THREAD

**Discussion by Qwakr45** [simple-login/app#1868] [Feature Requests]
Hello!

As I contacted SL support they told me, that for now reverse aliasing with BCC can’t work.

Proof:
![IMG_0048](https://github.com/simple-login/app/assets/144239514/532a8e6d-7725-41d6-aae5-12146b4b2a5a)

I think it would be great, if user would use BCC SL should send a completely new email (to hide fact of coping in headers). It will be really great for privacy.

**Comment by D1stract1on** [comment]
So… I thought it just works… I saw a post on Reddit, but… I think it is a misunderstanding.

@nguyenkims, can you explain this to us? 😅

**Comment by nguyenkims** [comment]
We've added a pull request to make BCC work, should be available tomorrow, can you please retry by then?
